MAINTAINERS.yml

```diff
@@ -3180,9 +3180,13 @@ Intel Platforms (X86):
     - tbursztyka
     - laurenmurphyx64
   files:
-    - boards/x86/
+    - boards/intel/intel_adl/
+    - boards/intel/intel_ehl/
+    - boards/intel/intel_rpl/
```

Comment:
why not `boards/intel` ?

although anyone can add an intel board, then I believe @jhedberg and others maintaining intel platforms would like to be noticed and added as reviewer.

Just like in Zephyr main, Nordic is added whenever a `*nrf*` board is modified, removed, or added:
https://github.com/zephyrproject-rtos/zephyr/blob/793c507209eb508c1c29441b56d60303df4187a7/MAINTAINERS.yml#L3395

Reply:
Because Intel splits platforms per arch in the maintainers file, see `Intel Platforms (X86)`, `Intel Platforms (Xtensa)`, etc. However, all intel boards will now be under `boards/intel`, regardless of their arch or platform. We probably need one more folder level in here, so maintainers file can be simplified.